ci/verify_env.jl

```diff
@@ -0,0 +1,40 @@
+"""
+The script checks, if custom dependency for unit tests via `CI_UNIT_PKG_URL_<package_name>` are set.
+If yes, the script fails, which means in practice, that the PR rely on non merged code.
```

Comment:
The script checks, if a custom dependency for unit tests via `CI_UNIT_PKG_URL_<package_name>` is set.
If so, the script fails, which means in practice, that the PR relies on non-merged code.